script/foreman-rake

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash
 USERNAME=$(/usr/bin/id -un)
 RAKE_CMD=/usr/bin/rake
+BUNDLE=bundle
 BUNDLER_CMD=""
 
 die() {
   echo $*; exit 1
 }
 
 cd ~foreman || die "Cannot change to foreman home directory"
-[ -f Gemfile ] && BUNDLER_CMD="bundle exec"
+[ -h /usr/bin/foreman-ruby ] && BUNDLE="/usr/bin/foreman-ruby /usr/bin/bundle"
```

Comment:
Instead of adding this line to the script itself, can the package build set BUNDLER_CMD using sed, i.e. `BUNDLER_CMD="/usr/bin/foreman-ruby /usr/bin/bundle exec"`? The RPM build already does something similar.

It'd need a small change to the `[ -f Gemfile ]` line not to change it if already set, but would keep the custom paths all within the packaging.